src/parsec.rs

```diff
@@ -182,7 +186,7 @@ fn create(full_id: FullId, gen_pfx_info: &GenesisPfxInfo) -> Parsec {
             *gen_pfx_info.first_info.hash(),
             full_id,
             &gen_pfx_info.first_info.members(),
-            vec![],
+            related_info,
```

Comment:
Why not take the related_info from the `gen_pfx_info.first_state_serialised` instead of passing it as argument?

Reply:
Yes, left over from an earlier attempt.